packages/botonic-cli/templates/intent/package.json

```diff
@@ -31,6 +31,7 @@
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-add-module-exports": "^1.0.2",
+    "chokidar": "^3.4.2",
```

Comment:
I think it's a pain having to maintain so many templates.
Let's think about it when we refactor them

Reply:
yes, I have created a task for this for next iterations of Botonic